app/components/Table.js

```diff
@@ -225,7 +250,13 @@ const Cell = styled.td`
   }
 `;
 
+const CellWrapper = styled(Flex)`
+  margin: 4px;
```

Comment:
This doesn't seem to be an improvement tbh 😬

Reply:
Not necessarily the best approach, but made the tables readable on mobile. 

<table>
	<tr> 
		<th> Before on mobile </th>
		<th> After on mobile </th>
	</tr>
<tr>
<td> <img src="https://user-images.githubusercontent.com/40148448/129172591-7c1f2b9a-f229-4b21-ba82-74f9841cc0a2.png"/>
</td>
<td> <img src="https://user-images.githubusercontent.com/40148448/129172597-391a4964-61a8-496d-a5b1-63e81282dde2.png"/>
</td> 
</tr>
</table>

Reply:
Gotcha, I still think the `ListItem` approach might be more appropriate for the volume of items we'll have in this list  anyway, it's very unlikely to ever reach more than 30 (1 per day for a month), which is two pages worth of results.

Reply:
Ah, sure. Let's do it with `ListItem` 😄

Reply:
Could pull the `Table` improvements into a separate PR potentially